src/Microsoft.Azure.WebJobs.Host/ConfigurationUtility.cs

```diff
@@ -34,6 +35,12 @@ public static void SetConfigurationFactory(Func<IConfiguration> configurationRoo
             Reset();
         }
 
+        public static bool IsDisabledMethod(string methodName)
+        {
+            var disabledSetting = GetSetting(string.Format(CultureInfo.InvariantCulture, "{0}.Disabled", methodName));
```

Comment:
All our other settings have the standard "AzureWebJobs" prefix (e.g. the Prefix defined in AmbientConnectionStringProvider). Should we do the same? E.g. `AzureWebJobs.MyFunction.Disabled`